internal/server/organizations.go

```diff
@@ -66,3 +66,28 @@ func (a *API) CreateOrganization(c *gin.Context, r *api.CreateOrganizationReques
 func (a *API) DeleteOrganization(c *gin.Context, r *api.Resource) (*api.EmptyResponse, error) {
 	return nil, access.DeleteOrganization(c, r.ID)
 }
+
+func (a *API) UpdateOrganization(c *gin.Context, r *api.UpdateOrganizationRequest) (*api.Organization, error) {
+	org, err := access.GetOrganization(c, r.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	// overwrite the existing domains to the incoming ones
+	org.AllowedDomains = []string{}
```

Comment:
A nice layer of security other tools do for this is _only_ allow domains that existing users have emails for. This avoids adding a random domain on accident. Should we consider that here?

Reply:
This creates a bit of a weird flow where you need to invite a user before you can allow their domain. Happy to re-visit this though.